service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -179,32 +181,62 @@ private String prepareReportLevelNarrativeStatements(DiagnosticReport diagnostic
         if (diagnosticReport.hasStatus()) {
             String status = PREPENDED_TEXT_FOR_STATUS + diagnosticReport.getStatus().toCode();
             String statusNarrativeStatement = buildNarrativeStatementForDiagnosticReport(
-                diagnosticReport, LABORATORY_RESULT_COMMENT.getCode(), status);
+                diagnosticReport.getIssuedElement(), LABORATORY_RESULT_COMMENT.getCode(), status);
 
             reportLevelNarrativeStatements.append(statusNarrativeStatement);
         }
 
         buildNarrativeStatementForMissingResults(diagnosticReport, reportLevelNarrativeStatements);
+        buildNarrativeStatementForObservationTimes(observations, reportLevelNarrativeStatements, diagnosticReport.getIssuedElement());
 
         return reportLevelNarrativeStatements.toString();
     }
 
+    private void buildNarrativeStatementForObservationTimes(List<Observation> observations, StringBuilder reportLevelNarrativeStatements,
```

Comment:
It's better to have all on the same line
```
private void buildNarrativeStatementForObservationTimes(
    List<Observation> observations, StringBuilder reportLevelNarrativeStatements, InstantType diagnosticReportIssued) {
```
or each on a separate:
```
private void buildNarrativeStatementForObservationTimes(
        List<Observation> observations, 
        StringBuilder reportLevelNarrativeStatements,
        InstantType diagnosticReportIssued) {
    ...
```
Notice the 8 spaces indent in the second example - to better distinguish arguments from function body